controlpanel/frontend/jinja2/tools.html

```diff
@@ -33,10 +33,13 @@ <h1 class="govuk-heading-xl">Your tools</h1>
             {% endfor %}
           {% endif %}
         {% else %}
-          {% if True or is_deploying[tool.name] %}
-          Deploying
+          {% if True %}
```

Comment:
won't this always be `True`?

Reply:
Yes, this is just temporary while I work on the functionality behind it